CONTRIBUTING.md

```diff
@@ -1,15 +1,13 @@
 CosmosOS is a volunteer effort. We encourage you to pitch in. Join the team!
 
-- Before you begin work, make sure to clear it with the project owners. It will save a lot of time down the road.
+- Before you begin work, make sure to discuss it with the maintainers here on github or on discord. It will save a lot of time down the road.
```

Comment:
```suggestion
- Before you begin work, make sure to discuss your plans with the maintainers here on GitHub or on [Discord](https://discord.gg/JDd9usuUgS). It will save a lot of time down the road.
```

Reply:
oops, already merged ^^'>